Firmware/fsensor.cpp

```diff
@@ -207,7 +207,7 @@ void fsensor_init(void)
 	}
 	printf_P(PSTR("FSensor %S"), (fsensor_enabled?PSTR("ENABLED"):PSTR("DISABLED")));
 #ifdef IR_SENSOR_ANALOG
-     printf_P(PSTR(" (sensor board revision:%S)\n"), (oFsensorPCB==ClFsensorPCB::_Rev04) ? _T(MSG_04_OR_NEWER) : _T(MSG_03_OR_OLDER));
+     printf_P(PSTR(" (sensor board revision:%S)\n"), ((oFsensorPCB==ClFsensorPCB::_Undef) ? _T(MSG_IR_UNKNOWN): ((oFsensorPCB==ClFsensorPCB::_Rev04) ? _T(MSG_IR_04_OR_NEWER) : _T(MSG_IR_03_OR_OLDER))));
```

Comment:
Can we do this relatively complex version text selection at one place? There are similar cases in ultralcd.cpp.
something like:
`const char *FsensorIRVersionText(){
switch(oFsensorPCB){
case oFsensorPCB==ClFsensorPCB::_Undef: return _T(MSG_IR_UNKNOWN);
....
}...`

Reply:
Fixed by @leptun